package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gatsby-source-contentstack",
-  "version": "5.0.4",
+  "version": "5.0.5",
```

Comment:
update the version and keep it separate from the master version @Contentstack-AnkitaD